ApplyToBecomeInternal/ApplyToBecome.Data/Models/AdvisoryBoardDecision/AdvisoryBoardDecision.cs

```diff
@@ -11,6 +11,10 @@ public class AdvisoryBoardDecision
 		public bool? ApprovedConditionsSet { get; set; }
 		public string ApprovedConditionsDetails { get; set; }
 		public List<AdvisoryBoardDeclinedReasons> DeclinedReasons { get; set; }
+		public string DeclineFinanceReason { get; set; }
```

Comment:
The model on the back end is going to look a bit different to this. 
@MartinWheelerMT said he is going to change the AdvisoryBoardDeclinedReasons class so that it has a Details property. That would be the best place to put the reason details.
@MartinWheelerMT please can you share the diagram you put together.

Reply:
![image](https://user-images.githubusercontent.com/109524605/184141850-224a9f1a-632d-475b-8331-4320fdd63226.png)

Reply:
Yeah; the API-side changes came about from a discussion after realising that this code would need them.